modules/argocd-htpasswd-sample.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-htpasswd-sample_{context}"]
+= HTPasswd OAuth provider
+
+You can use ArgoCD to deploy an HTPasswd OAuth provider. The following sample defines and ArgoCD application that you can use to log in to your cluster as the `user1` user with the password `MyPassword!`.
+
+.ArgoCD HTPassword OAuth provider sample (`oauth-htpasswd.yaml`)
+----
+---
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+  annotations:
+    argocd.argoproj.io/sync-options: Prune=false
+spec:
+  identityProviders:
+  - name: my_identity_provider
+    mappingMethod: claim
+    type: HTPasswd
+    htpasswd:
+      fileData:
+        name: htpass-secret
+----
+
+.Secret for the HTPassword OAuth provider sample (`htpass-secret.yaml`)
+----
+---
+apiVersion: v1
+data:
+  htpasswd: dXNlcjE6JDJ5JDA1JFBqUWVqZ1VNMzU5LnJkOE5xQzlweXVHbk9JM3JrVWl4cWh2NDg4enJVNVJvaEhZa2dLWGFhCg==
+kind: Secret
+metadata:
+  name: htpass-secret
+  namespace: openshift-config
+  annotations:
+    argocd.argoproj.io/sync-options: Prune=false
```

Comment:
This won't be needed as pruning it's not being enabled when creating the Argo CD App.